scripts/generate_rdf_docs.py

```diff
@@ -111,7 +111,7 @@ def markdown_from_doc(doc: DocNode, indent: int = 0):
 
             print("name", name)
 
-        name = f"`{name}` " if name else ""
+        name = f'<a name="{name}"></a>`{name}` ' if name else ""
```

Comment:
Maybe you can try `id=`, see here: https://stackoverflow.com/a/484781

Reply:
thanks @oeway let's give that a try...

Reply:
works like a charm!